src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule711generalform/InputFormattedSingleLineJavadocAndInvalidJavadocPosition.java

```diff
@@ -0,0 +1,212 @@
+package // violation 'package statement should not be line-wrapped.'
+    /** invalid javadoc. */
+    // violation above 'Javadoc comment is placed in the wrong location.'
+    com.google.checkstyle.test.chapter7javadoc.rule711generalform;
+
+/** invalid javadoc. */
+// violation above 'Javadoc comment is placed in the wrong location.'
+import javax.swing.JFrame;
+
+/** some javadoc. */
+public class InputFormattedSingleLineJavadocAndInvalidJavadocPosition {
+
+  /** As of JDK 1.1, replaced by {@link #setBounds(int,int,int,int)}. */
+  void foo1() {}
+
+  /** As of JDK 1.1, replaced by {@link #setBounds(int,int,int,int)}. */
+  void foo2() {}
+
+  /**
+   * @throws CheckstyleException if a problem occurs
+   */
+  // violation 3 lines above 'Summary javadoc is missing.'
+  void foo3() {}
+
+  /**
+   * summary.
+   *
+   * @throws CheckstyleException if a problem occurs
+   */
+  void foo4() {}
+
+  /** An especially short bit of Javadoc. */
+  void foo5() {}
+
+  /** An especially short bit of Javadoc. */
+  void foo6() {}
+
+  /**
+   * @inheritDoc
+   */
+  // violation 3 lines above 'Summary javadoc is missing.'
+  void foo7() {}
+
+  /** {@inheritDoc} */
+  void foo8() {}
+
+  /**
+   * @customTag
+   */
+  // violation 3 lines above 'Summary javadoc is missing.'
+  void bar() {}
+
+  /**
+   * summary.
+   *
+   * <h1>Some header </h1>
+   *
+   * {@inheritDoc} {@code bar1} text
+   */
+  void bar2() {}
+
+  /**
+   * @customTag <a> href="https://github.com/checkstyle/checkstyle/"</a>text
+   */
+  // violation 3 lines above 'Summary javadoc is missing.'
+  void bar3() {}
+
+  /** Single line Javadoc that references {@link String}. */
+  void bar4() {}
+
+  /**
+   * @return in single line javadoc
+   */
+  // violation 3 lines above 'Summary javadoc is missing.'
+  int bar5() {
+    return 0;
+  }
+
+  /**
+   * summary.
+   *
+   * @return in multi line javadoc
+   */
+  int bar6() {
+    return 0;
+  }
+}
+
+/** invalid javadoc. */
+// violation above 'Javadoc comment is placed in the wrong location.'
+/** valid javadoc. */
+class ExtraInputInvalidJavadocPosition {
```

Comment:
original code is:
```
/** invalid javadoc. */ // violation 'Javadoc comment is placed in the wrong location.'
/** valid javadoc. */
class InputInvalidJavadocPosition {
```

please change original code  (rule711generalform/InputSingleLineJavadocAndInvalidJavadocPosition.java)
to be:
```
// violation below 'Javadoc comment is placed in the wrong location.'
/** invalid javadoc. */ 
/** valid javadoc. */
class InputInvalidJavadocPosition {
```

please apply this pattern to all similar code in this file. It will simply a diff AND will put marker comment a bit outside of target code to not contribute to it weirdness by itself.